src/firefly/java/edu/caltech/ipac/firefly/server/visualize/fitseval/SpectralCubeEval.java

```diff
@@ -24,7 +24,7 @@
 class SpectralCubeEval implements FitsEvaluation.Eval {
     @Override
     public List<RelatedData> evaluate(File f, FitsRead[] frAry, BasicHDU[] HDUs, int fitsReadIndex, int hduIndex, WebPlotRequest req) {
-        if (!req.getBooleanParam("SpectralCube")) return null;
+        if (req!=null && !req.getBooleanParam("SpectralCube")) return null;
         // do spectral cube processing here
         // todo Convert FitsCube to work in this environment
```

Comment:
This `todo` represents a hook for future spectral cube work. When we are ready to do serious spectral cube evaluation, it will go here.